packages/adapter-node/src/libs/utils.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Parses the given value into number of bytes.
+ *
+ * @param {string} value - Size in bytes. Can also be specified with a unit suffix kilobytes (K), megabytes (M), or gigabytes (G).
+ */
+export function parseAsBytes(value) {
```

Comment:
We use snake case for internal variables
```suggestion
export function parse_as_bytes(value) {
```